projects/packages/connection/src/class-nonce-handler.php

```diff
@@ -176,16 +176,26 @@ public function delete( $limit = 10, $cutoff_timestamp = null ) {
 			)
 		);
 
-		if ( ! is_array( $ids ) || ! count( $ids ) ) {
-			// There's either nothing to remove, or there's an error and we can't proceed.
+		if ( ! is_array( $ids ) ) {
+			// There's an error and we can't proceed.
+			return false;
+		}
+
+		$ids = array_filter( $ids );
```

Comment:
Such a humble line to fix such an epic misconfiguration, don't you think? :) I believe this one deserves a comment about how there can be sites with all option IDs set to `0`,

Reply:
Done, thanks for pointing this out!